runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingModeExecutionContext.java

```diff
@@ -177,11 +190,15 @@ public void start(
       Work work,
       WindmillStateReader stateReader,
       SideInputStateFetcher sideInputStateFetcher,
+      long maxOutputKeyBytes,
```

Comment:
what about taking OperationalLimits instead of plumbing separately?

Reply:
Sounds good. We might want some additional fields later so that'd help with plumbing.